### DISCUSSION TITLE
PTC value for renewables

### DISCUSSION THREAD

**Discussion by wanyingw193** [NREL/ReEDS-2.0#105] [Q&A: Analysis]
Morning ReEDS team!

For the PTC value calculation of renewables, the formula obtained from Support_function.py is." PTC_unit_value = Ptc_value * (1-PTC_tax_equity_penalty) * 1/ (1-tax_rate) * 1/CRF" to generate the PTC_unit_value parameter in the objective function.

Do I understand correctly that the t for calculating the "CRF = r * (1+r)^t / (1+r)^t -1" is related to the duration of the PTC value? For example, 10 years for onshore wind.

Thanks!

**Comment by wesleyjcole** [comment]
The function cal_ptc_unit_value in support_functions.py shows the ptc_unit_value calulation, which explicitly includes the PTC duration in the calculation.

**Reply by wanyingw193** [reply]
Found it! I missed this before, thanks for the guidance!

**Comment by wanyingw193** [comment]
Hi NREL team, 
 
For the PTC_unit_value.csv input in the objective function, why does the ptc value occur only once for investment-capacity and not repeatedly over 10 years based on the PTC duration?

Which gams or python file applies itc_fraction.csv to calculate ITC?

Thank you!

**Comment by wesleyjcole** [comment]
The ReEDS objective function uses a 20-year net present value calculation, so we apply the PTC value as a present value capital cost adjustment in the objective function to sync things up.  This works because wind is at the bottom of the dispatch stack.

The itc_fraction isn't used directly in GAMS, but for post-processing.  The ITC fraction adjust the cost_cap_fin_mult, which is how the ITC is seen in the objective function.

**Reply by wanyingw193** [reply]
Thank you! A follow up question on PTC_unit_value.

There are two methods to calculate the obtained PTC credit for renewable energy with ptc_unit_value in the objective function (screenshot attached). 
Why are two different calculation methods needed? Why can't the technology with m_cf(i) be calculated using GEN?
![image](https://user-images.githubusercontent.com/78582696/187846498-a7ab2538-dcb5-4f7c-80c2-925907ec903e.png)

Thank you so much!

**Reply by wesleyjcole** [reply]
The PTC was originally built for wind, which has zero marginal cost, so it would generate irrespective of the PTC.  Other technologies (CCS) would not, requiring that the generation be included in the objective function.  These could be combined into a single consistent treatment, but that just hasn't happened yet.  We are revamping some of this treatment for the 2022 version of the model to better capture the Investment Reduction Act, so some of this may change there.

**Reply by wanyingw193** [reply]
Thank you!
